code/modules/research/xenobiology/crossbreeding/_clothing.dm

```diff
@@ -0,0 +1,131 @@
+	if(slot == SLOT_GLASSES)
+		return TRUE
+
+/obj/structure/light_prism
+	name = "light prism"
+	desc = "A shining crystal of semi-solid light. Looks fragile."
+	icon = 'icons/obj/slimecrossing.dmi'
+	icon_state = "lightprism"
+	density = FALSE
+	anchored = TRUE
+	max_integrity = 10
+
+/obj/structure/light_prism/Initialize(var/colour)
+	. = ..()
+	color = colour
+	light_color = colour
+	set_light(5)
+
+/obj/structure/light_prism/attack_hand(mob/user)
+	to_chat(user, "<span class='notice'>You dispel [src]</span>")
+	qdel(src)
+
+/datum/action/item_action/change_prism_colour
+	name = "Adjust Prismatic Lens"
+	icon_icon = 'icons/obj/slimecrossing.dmi'
+	button_icon_state = "prismcolor"
+
+/datum/action/item_action/change_prism_colour/Trigger()
+	var/obj/item/clothing/glasses/prism_glasses/glasses = target
+	var/new_color = input(owner, "Choose the lens color:", "Color change",glasses.colour) as color|null
+	if(!new_color)
+		return
+	glasses.colour = new_color
+
+/datum/action/item_action/place_light_prism
+	name = "Fabricate Light Prism"
+	icon_icon = 'icons/obj/slimecrossing.dmi'
+	button_icon_state = "lightprism"
+
+/datum/action/item_action/place_light_prism/Trigger()
+	var/obj/item/clothing/glasses/prism_glasses/glasses = target
+	if(locate(/obj/structure/light_prism) in get_turf(owner))
+		to_chat(owner, "<span class='warning'>There isn't enough ambient energy to fabricate another light prism here.</span>")
+		return
+	if(istype(glasses))
+		if(!glasses.colour)
+			to_chat(owner, "<span class='warning'>The lens is oddly opaque...</span>")
+			return
+		to_chat(owner, "<span class='notice'>You channel nearby light into a glowing, ethereal prism.</span>")
+		new /obj/structure/light_prism(get_turf(owner), glasses.colour)
+
+/obj/item/clothing/head/peaceflower
+	name = "heroine bud"
+	desc = "An extremely addictive flower, full of peace magic."
+	icon = 'icons/obj/slimecrossing.dmi'
+	icon_state = "peaceflower"
+	item_state = "peaceflower"
+
+/obj/item/clothing/head/peaceflower/equipped(mob/living/carbon/human/user, slot)
+	. = ..()
+	if(slot == SLOT_HEAD)
+		user.add_trait(TRAIT_PACIFISM, "peaceflower_[REF(src)]")
+
+/obj/item/clothing/head/peaceflower/dropped(mob/living/carbon/human/user)
+	..()
+	user.remove_trait(TRAIT_PACIFISM, "peaceflower_[REF(src)]")
+
+/obj/item/clothing/head/peaceflower/attack_hand(mob/user)
+	if(iscarbon(user))
+		var/mob/living/carbon/C = user
+		if(src == C.head)
+			to_chat(user, "<span class='warning'>You feel at peace. <b style='color:pink'>Why would you want anything else?</b></span>")
+			return
+	return ..()
+
+/obj/item/clothing/suit/armor/heavy/adamantine
+	name = "adamantine armor"
+	desc = "A full suit of adamantine plate armor. Impressively resistant to damage, but weighs about as much as you do."
+	icon_state = "adamsuit"
+	item_state = "adamsuit"
+	flags_inv = list()
+	obj_flags = IMMUTABLE_SLOW
+	var/hit_reflect_chance = 40
+	slowdown = 4
```

Comment:
If some dude eats some slowdown, someone's going to have a bad day

On the bright side this isn't a acid or fire proof piece of clothing that somewhat weak to lasers having the same protection values of deathsquad armor

Reply:
Definitely one of the tankier things players can access, especially with the laser reflection chance. If you take this with the adamantine shield, you won't be moving very fast, but you'll basically be immortal.